docs/history.md

```diff
@@ -84,6 +84,9 @@ Read our [Migration Guide](https://guide.meteor.com/2.7-migration.html) for this
 * `meteor-node-stubs@1.2.1`
   - Adds support for [node:](https://nodejs.org/api/esm.html#node-imports) imports.
 
+* `minifier-jss@2.8.0`
+  - Replacing 
```

Comment:
Missing swc?

Reply:
https://github.com/terser/terser/blob/master/CHANGELOG.md#v5121
>Fixed an issue with function definitions inside blocks (https://github.com/terser/terser/issues/1155)

Reply:
My bad... I did a commit without adding the whole message. Thank you for pointing it out!